content/200-concepts/100-components/03-prisma-migrate/100-legacy-migrate.mdx

```diff
@@ -3,11 +3,12 @@ title: 'Legacy Prisma Migrate'
 metaTitle: 'Legacy Prisma Migrate (Reference)'
 metaDescription: 'Legacy Prisma Migrate is a declarative data modeling and schema migration tool that is available via the Prisma CLI.'
 tocDepth: 2
+hidePage: true
 ---
 
 <TopBlock>
 
-> **Important!** This page documents legacy Prisma Migrate (Experimental) available in version 2.12.0 and earlier. [Prisma Migrate (Preview)](.) is available in version [2.13.0](https://github.com/prisma/prisma/releases/tag/2.13.0) and later.
+> **Important!** This page documents legacy Prisma Migrate (Experimental) available in version 2.12.0 and earlier. [Prisma Migrate)](.) is available in version [2.13.0](https://github.com/prisma/prisma/releases/tag/2.13.0) and later.
```

Comment:
`2.13.0` is still in Preview. Maybe we can add "and generally available since 2.19.0"?

Reply:
Done!